extensions/ftp/src/main/java/org/mule/extension/ftp/internal/ftp/connection/ClassicFtpConnectionProvider.java

```diff
@@ -44,58 +52,35 @@
     private String password;
 
     /**
-     * Creates and returns a new instance of {@link FtpFileSystem}
-     *
-     * @param config the {@link FtpConnector} which parametrizes the return value
-     * @return a {@link FtpFileSystem}
+     * The transfer mode to be used. Currently {@code BINARY}
+     * and {@code ASCII} are supported.
+     * <p>
+     * Defaults to {@code BINARY}
      */
-    @Override
-    public FtpFileSystem connect(FtpConnector config) throws ConnectionException
-    {
-        return new FtpFileSystem(config, createClient(config));
-    }
+    @Parameter
+    @Optional(defaultValue = "BINARY")
```

Comment:
Java annotations don't allow that. This is the only way of doing it, SDK will do the proper transformations transparently